get-started/part3.md

```diff
@@ -103,6 +103,19 @@ This `docker-compose.yml` file tells Docker to do the following:
 - Define the `webnet` network with the default settings (which is a
   load-balanced overlay network).
 
+Notice that we've set the Compose file version to version 3. This essentially
+makes it [swarm mode](/engine/swarm/index.md) compatible. We can make use of the
```

Comment:
I Agree, this little explanation is less interesting for someone who is just starting to use docker, but for someone who had some experience with older docker versions this is going to be very helpful.